src/commands/install.ts

```diff
@@ -450,6 +451,9 @@ export async function install(
     sourcemap: sourceMap,
     exports: 'named',
     chunkFileNames: 'common/[name]-[hash].js',
+    plugins: [
+      ...(config.buildOptions.optimize ? [terser({compress: false, mangle: true})] : []), // https://github.com/terser/terser#terser-fast-minify-mode
+    ],
```

Comment:
Note this is in `outputOptions` in Rollup—after we do all the Snowpack magic. Overall, a pretty safe add (it only adds a _little_ increase to the install & build time, but it’s not noticeable on my machine, and can be opted out-of by users).